app/webpacker/components/wca/FormBuilder/input/MarkdownEditor.js

```diff
@@ -100,8 +108,8 @@ function getOptions(disabled) {
     spellChecker: false,
     promptURLs: true,
     previewRender,
-    status: ['upload-image'],
-    uploadImage: true,
+    status: [],
+    uploadImage: false,
```

Comment:
You can directly pass `editorOptions.enableImageUpload` here

Reply:
Done